litgpt/deploy/serve.py

```diff
@@ -15,6 +16,13 @@
 from litgpt.utils import load_checkpoint, CLI, get_default_supported_precision
 
 
+_LITSERVE_AVAILABLE = RequirementCache("litserve")
+if _LITSERVE_AVAILABLE:
+    from litserve import LitAPI, LitServer
+else:
+    LitAPI, LitServer = object, object
```

Comment:
LitServer can be a local import in the main block

Reply:
But isn't it more readable to have it all in one place?

Reply:
I wouldn't say so. Up to you